packages/framework/presence/src/datastorePresenceManagerFactory.ts

```diff
@@ -8,12 +8,24 @@
  */
 
 import type { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { assert } from "@fluidframework/core-utils/internal";
+import type { IInboundSignalMessage } from "@fluidframework/runtime-definitions/internal";
 import type { SharedObjectKind } from "@fluidframework/shared-object-base";
 
 import { BasicDataStoreFactory, LoadableFluidObject } from "./datastoreSupport.js";
 import type { IPresence } from "./presence.js";
 import { createPresenceManager } from "./presenceManager.js";
 
+import type { IExtensionMessage } from "@fluid-experimental/presence/internal/container-definitions/internal";
+
+function assertSignalMessageIsValid(
+	message: IInboundSignalMessage | IExtensionMessage,
+): asserts message is IExtensionMessage {
+	assert(message.clientId !== null, "Signal must have a client ID");
```

Comment:
Is this source or target clientID? Self-descriptive names are usually helpful :)

Reply:
That is the long standing id for the client that sent the message. Renaming is not going to happen. A message with a target client id uses the property name `targetClientId`.